src/test/util_test.ts

```diff
@@ -46,9 +46,14 @@ describe('util', () => {
         type: 'ObjectExpression',
         properties: []
       };
-      const entry = util.extractPropertyEntry(node);
+      const key: ESTree.Identifier = {
```

Comment:
can you add one of these tests for a non-identifier?

e.g. `should extract property config with non-identifier key`

you could use a `Literal` for example, like:

```ts
const key: ESTree.Literal = {
  type: 'Literal',
  value: 'foo',
  raw: '\'foo\''
};
```